### PR TITLE
W 7pm b disp deadlines

### PR DESCRIPTION
**Pull request by brandonkatz**
Added a table to personal schedule page that displays important dates regarding registration.

### CODE REVIEW

**Comment by andrewhlu** [comment]
@brandonkatz Just an FYI: you have some failing tests and a merge conflict with the master branch. Please resolve these before requesting our review for your changes.

File: src/main/java/edu/ucsb/cs56/ucsb_courses_search/service/UCSBQuarterCalendarService.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+@Service
+public class UCSBQuarterCalendarService implements CalendarService {
+
+    private Logger logger = LoggerFactory.getLogger(UCSBQuarterCalendarService.class);
+
+    private String apiKey;
+
+    public UCSBQuarterCalendarService(@Value("${ucsb.api.consumer_key}") String apiKey) {
+        this.apiKey = apiKey;
+        logger.info("apiKey=" + apiKey);
```
Comment (agprice) [inline src/main/java/edu/ucsb/cs56/ucsb_courses_search/service/UCSBQuarterCalendarService.java]: This log statement absolutely needs to be removed. This publicly leaks the API key. You need to reset your UCSB API key. Never log any secret key. I know this was done elsewhere in the code, but it's a huge bug and security risk.

**Comment by agprice** [comment]
This PR seem to reintroduce https://github.com/ucsb-cs56-w20/ucsb-courses-search/issues/302

**Comment by agprice** [comment]
The formatting of the final schedules seems broken with the latest update.

**Pull request merged**